src/components/shared/Article/ArticleSecondaryCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Grid } from 'semantic-ui-react';
+import ArticleUtil from '../../../utils/articles';
+
+const { parseArticleCreationDate } = ArticleUtil;
+
+const StyledColumn = styled(Grid.Column)`
+  text-align: center;
+`;
+
+const StyledNumber = styled.p`
+  font-size: 1.8em;
+  color: #aaa;
+`;
+
+const ArticleAuthor = styled.p`
+  padding: 0;
+  margin: 0;
+`;
+
+const ArticleSecondaryCard = ({ article, count }) => (
+    <Grid.Row>
+      <StyledColumn width={4} verticalAlign="middle">
+        <StyledNumber>0{count + 1}</StyledNumber>
+      </StyledColumn>
+
+      <Grid.Column>
+        <h3>{article.title}</h3>
```

Comment:
Could you have considered deep destructuring? @micah-akpan

Reply:
Okay that's a good suggestion. I just didn't see the need at the time. I will incorporate the suggested changes. Thanks